core/index.js

```diff
@@ -80,7 +80,7 @@ export async function createApp(projectConfig, env) {
   // load initial url and initial state if host provided it
   const host = useHost();
   const initialState = host.getState();
-  const context = host.getMeta();
+  const context = { ...host.getMeta(), 'configuration': host.getConfiguration() };
```

Comment:
quotes around property name not needed

```suggestion
  const context = { ...host.getMeta(), configuration: host.getConfiguration() };
```